src/commands/utils_command/sign_transaction_with_ledger_subcommand/mod.rs

```diff
@@ -0,0 +1,94 @@
+use dialoguer::Input;
+use near_primitives::borsh::BorshSerialize;
+use std::str::FromStr;
+
+/// Utility to sign transaction on Ledger
+#[derive(Debug, Default, clap::Clap)]
+pub struct CliSignTransactionWithLedger {
+    #[clap(long)]
+    seed_phrase_hd_path: Option<slip10::BIP32Path>,
+    #[clap(long)]
+    unsigned_transaction: Option<crate::common::TransactionAsBase64>,
+}
+
+#[derive(Debug)]
+pub struct SignTransactionWithLedger {
+    pub seed_phrase_hd_path: slip10::BIP32Path,
+    pub unsigned_transaction: near_primitives::transaction::Transaction,
+}
+
+impl From<CliSignTransactionWithLedger> for SignTransactionWithLedger {
+    fn from(item: CliSignTransactionWithLedger) -> Self {
+        let seed_phrase_hd_path = match item.seed_phrase_hd_path {
+            Some(hd_path) => hd_path,
+            None => slip10::BIP32Path::from_str("44'/397'/0'/0'/1'").unwrap(),
```

Comment:
I believe we should let users enter this custom value in interactive mode. Let's add a method similar to `input_unsigned_transaction` (prepopulate the input with this default value)

Reply:
Disagree, users who will use interactive mode in most cases will use the default hd path. The ones who really need to specify custom hd path will pass it via parameter of command line. WDYT?

Reply:
I considered that as well. Let me share my thinking:

1. It is easy to drop into interactive mode, and you have to make an explicit choice to opt-out of it and ask for `--help`.
2. If you don't see the help, you won't learn about HD path existance, and sometimes you actually need to change it (if you store more than a single key/account)
3. If you need to modify it at any point, you will be forced to craft the whole command, and interactive mode won't be an option for you anymore

Thus, I believe having an option to edit hd path is a good thing. We may just write a good message like "if you are not sure, keep the default" (that is why I proposed to pre-populate the value)

Reply:
Makes sense. I'll update the code asap